openeogeotrellis/deploy/run_graph_locally.py

```diff
@@ -19,6 +19,7 @@ def run_graph_locally(process_graph, output_dir):
     process_graph = as_flat_graph(process_graph)
     if "process_graph" not in process_graph:
         process_graph = {"process_graph": process_graph}
+    files_orig = set(output_dir.glob("*"))
```

Comment:
Does this glob work with files not in this output directory e.g. `filepath_band_band`?

Reply:
Good question, I'll test

Reply:
A fix was needed indeed, Thanks!